addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -486,7 +486,7 @@ export class Wysiwyg extends Component {
             plugins: options.editorPlugins,
             direction: options.direction || localization.direction || 'ltr',
             collaborationClientAvatarUrl: this._getCollaborationClientAvatarUrl(),
-            renderingClasses: ['o_dirty', 'o_transform_removal', 'oe_edited_link', 'o_menu_loading'],
+            renderingClasses: ['o_dirty', 'o_transform_removal', 'oe_edited_link', 'o_menu_loading', "o_draggable"],
```

Comment:
Good commit message :+1: 

Now, with your changes, when a code instruction adds or removes the `o_draggable` class, it will never do a history step right ? If yes, you can remove the `observerUnactive`/`observerActive` lines that QSM added no ?

Also, I know it's annoying that we changed the `"`/`'` rule but I think it's weird to have a mix of the two rules on the same line. Apply the old or the new rule on the entire line :)